### PR TITLE
#284 made end timestamp respond to delay switch also fix for #273

### PR DESCRIPTION
**Pull request by vinayakathavale**

### CODE REVIEW

**Comment by abhibansal530** [comment]
But it does not solves the issue of timestamp being not incremented.

**Comment by abhishek-vinjamoori** [comment]
@abhibansal530. It's just a partial fix to the whole issue.

**Comment by vinayakathavale** [comment]
@canihavesomecoffee  can you run regression tests on this one 
Thanks

File: src/lib_ccx/stream_functions.c [hunk 1]
```diff
@@ -94,14 +94,14 @@ void detect_stream_type (struct ccx_demuxer *ctx)
...
+		if (boxScore > 7) // will check again for mp4 with boxscore > 1 after checking for all other file types
 		{
 			// We had at least one box (or multiple) at the end to "claim" this is MP4. A single valid box at the end is doubtful...
 			ctx->stream_mode = CCX_SM_MP4;
 		}
 	}
 	if (ctx->stream_mode==CCX_SM_ELEMENTARY_OR_NOT_FOUND) // Still not found
-	{
+	{	
 		if (ctx->startbytes_avail > 188*8) // Otherwise, assume no TS
```
Comment (anshul1912) [inline src/lib_ccx/stream_functions.c]: Please avoid whitespace.

File: src/lib_ccx/stream_functions.c [hunk 2]
```diff
@@ -195,7 +228,7 @@ int detect_myth( struct ccx_demuxer *ctx )
 {
 	int vbi_blocks=0;
 	// VBI data? if yes, use myth loop
-	// STARTBTYTESLENGTH is 1MB, if the file is shorter we will never detect
+	// STuARTBTYTESLENGTH is 1MB, if the file is shorter we will never detect
 	// it as a mythTV file
```
Comment (anshul1912) [inline src/lib_ccx/stream_functions.c]: why STuart start was looking much better ;)

**Comment by anshul1912** [comment]
@vinayakathavale : there is an option -f for git push, if you already know the mistake that you have done, I would suggest better to git reset HEAD^  then update your changes then commit it.
while pushing use git push -f

File: src/lib_ccx/stream_functions.c (hunk 1 above)
Comment (vinayakathavale) [inline src/lib_ccx/stream_functions.c]: sorry will change that

**Comment by canihavesomecoffee** [comment]
@vinayakathavale Will do, it escaped my attention...

@ccx-testerbot runtests master

**Comment by canihavesomecoffee** [comment]
Doesn't look to break more than what is currently broken, so :+1:

**Pull request closed without merging**